src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -2867,6 +2871,8 @@ NEVER_INLINE Value ByteCodeInterpreter::openLexicalEnvironment(ExecutionState*&
     bool shouldUseHeapAllocatedState = true;
     if (inPauserResumeProcess) {
         shouldUseHeapAllocatedState = false;
+    } else if (code->m_kind == OpenLexicalEnvironment::ClassStaticFieldInit) {
+        shouldUseHeapAllocatedState = false;
```

Comment:
How about merging these two conditional statements into one if statement?

Reply:
That's good idea

Reply:
I fixed it